### PR TITLE
Add Web App Manifest Application Information specification

### PR DESCRIPTION
**Pull request by filips123**
The `categories`, `description`, `iarc_rating_id` and `screenshots` members of [Web App Manifest](https://w3c.github.io/manifest/) have been moved into new [Web App Manifest - Application Information](https://w3c.github.io/manifest-app-info/) specification.

I will also soon create a separate PR to https://github.com/mdn/content to update needed spec links.

### CODE REVIEW

File: kumascript/macros/SpecData.json [hunk 1]
```diff
@@ -1099,6 +1099,11 @@
     "url": "https://w3c.github.io/manifest/",
     "status": "WD"
   },
+  "ManifestAppInfo": {
+    "name": "Web App Manifest - Application Information",
+    "url": "https://w3c.github.io/manifest-app-info/",
+    "status": "NOTE"
```
Comment (escattone) [inline kumascript/macros/SpecData.json:1105]:
```suggestion
    "status": "ED"
```

**Review by escattone — CHANGES_REQUESTED** [review]

**Comment by filips123** [comment]
I applied requested changes, but it seems this PR does not need to be merged anymore because of mdn/content#5164.

File: kumascript/macros/SpecData.json (hunk 1 above)
Comment (escattone) [reply kumascript/macros/SpecData.json]: It seems like this is an `Editor's Draft` rather than a `Note`?

**Review by Elchi3 — APPROVED** [review]
Thanks! It doesn't hurt to merge this. We haven't fully transitioned away from this file yet.

**Review by escattone — APPROVED** [review]

**Comment by escattone** [comment]
Thanks @Elchi3!

**Pull request merged**